benchexec/tablegenerator/react-table/src/workers/scripts/stats.worker.js

```diff
@@ -0,0 +1,392 @@
+    }
+    if (!totalNaNInfo.hasNaN) {
+      total.sum = maybeAdd(total.sum, column, type);
+      updateMaxDecimalMetaInfo({ columnType, column, bucket: total });
+    }
+
+    if (!isNaN(Number(column))) {
+      if (!skipBucket) {
+        bucket.max = mathStringMax(bucket.max, column);
+        bucket.min = mathStringMin(bucket.min, column);
+      }
+      if (!skipSubTotal) {
+        subTotalBucket.max = mathStringMax(subTotalBucket.max, column);
+        subTotalBucket.min = mathStringMin(subTotalBucket.min, column);
+      }
+    }
+    if (!skipBucket) {
+      try {
+        bucket.items.push(item);
+      } catch (e) {
+        console.e({ bucket, bucketMeta: bucketNaNInfo, key });
+      }
+    }
+    if (!skipSubTotal) {
+      try {
+        subTotalBucket.items.push(item);
+      } catch (e) {
+        console.e({ subTotalBucket, bucketMeta: bucketNaNInfo, totalKey });
+      }
+    }
+
+    buckets[key] = bucket;
+    buckets[totalKey] = subTotalBucket;
+  }
+
+  for (const [bucket, values] of Object.entries(buckets)) {
+    if (shouldSkipBucket(bucketNaNInfo, bucket)) {
+      continue;
+    }
+    calculateMean(values, values.items);
+
+    calculateMedian(values, values.items);
+    buckets[bucket] = values;
+  }
+  const totalHasNaN = totalNaNInfo.hasNaN;
+
+  if (totalHasNaN) {
+    total = { ...total, ...nanObj };
+  } else {
+    calculateMean(total, copy);
+    calculateMedian(total, copy);
+  }
+
+  for (const item of copy) {
+    const { column } = item;
+    if (isNaN(Number(column))) {
+      continue;
+    }
+    const numCol = Number(column);
+    const key = `${item.categoryType}-${item.resultType}`;
+    const totalKey = `${item.categoryType}-total`;
+    const bucket = buckets[key];
+    const subTotalBucket = buckets[totalKey];
+    const diffBucket = numCol - bucket.avg;
+    const diffSubTotal = numCol - subTotalBucket.avg;
+    const diffTotal = numCol - total.avg;
+    total.variance += Math.pow(diffTotal, 2);
+    bucket.variance += Math.pow(diffBucket, 2);
+    subTotalBucket.variance += Math.pow(diffSubTotal, 2);
+  }
+
+  const totalHasNegInf = Number(total.min) === -Infinity;
+  const totalHasPosInf = Number(total.max) === Infinity;
+  total.stdev = calculateStdev(
+    totalHasNegInf,
+    totalHasPosInf,
+    total.variance,
+    copy.length,
+  );
+
+  for (const [bucket, values] of Object.entries(buckets)) {
+    if (shouldSkipBucket(bucketNaNInfo, bucket)) {
+      for (const [key, val] of Object.entries(values)) {
+        values[key] = val.toString();
+      }
+      buckets[bucket] = values;
+      continue;
+    }
+    const valuesHaveNegInf = Number(values.min) === -Infinity;
+    const valuesHavePosInf = Number(total.max) === Infinity;
+    values.stdev = calculateStdev(
+      valuesHaveNegInf,
+      valuesHavePosInf,
+      values.variance,
+      values.items.length,
+    );
+
+    for (const [key, val] of Object.entries(values)) {
+      if (key === "meta") {
```

Comment:
What is the meaning of `meta` here? It is not a common term in BenchExec, and it did not find documentation. Because the term is so vague we might also want to rename it (depending on what it is used for).

Reply:
`meta` stores meta-information (like `maxDecimals` and `type`) of the current bucket, which will be used by the number formatter. 
I will add additional documentation.